roles/kubernetes/preinstall/tasks/main.yml

```diff
@@ -78,7 +74,6 @@
   command: dnf install -y python-dnf yum
   when: ansible_distribution == "Fedora" and
         ansible_distribution_major_version > 21
-  changed_when: False
```

Comment:
@ant31 it would be great to get notified when this ansible script has installed another version of a package, don't you think so?